lib/diagnostic.rb

```diff
@@ -12,8 +12,14 @@
 # Name should be readable and writeable, age should only be readable,
 # and location should only be writable.
 
+
+
 ##
-# your code here
+class Person
+  def initialize(name,age,location)
+    @name = name
+    @age = age
+    @location = location
```

Comment:
You need to att `attr_reader` `attr_writer` and `attr_accessor` to have the setters and getters for later on.